config/locales/en/placements/schools/users.yml

```diff
@@ -8,14 +8,15 @@ en:
           change_organisation: Change organisation
         show:
           change_organisation: Change organisation
-          remove_user: Remove user
+          delete_user: Delete user
         destroy:
           user_removed: User removed
         remove:
-          page_title: Are you sure you want to remove this user? - %{user_name}
-          are_you_sure: Are you sure you want to remove this user?
-          remove_user: Remove user
+          page_title: Are you sure you want to delete this user? - %{user_name}
+          are_you_sure: Are you sure you want to delete this user?
+          delete_user: Delete user
           cancel: Cancel
           change_organisation: Change organisation
-          user_will_be_sent_an_email: The user will be sent an email to tell them you removed them from %{organisation_name}
+          user_will_no_longer: "%{user_name} will no longer be able to view, create or manage placements at your school."
+          user_will_be_sent_an_email: The user will be sent an email to tell them you deleted them from %{organisation_name}
         you_cannot_perform_this_action: You cannot perform this action
```

Comment:
I think this can be removed.

Reply:
Hmm the design says to update the warning text from removed -> deleted.